omnibus/files/private-chef-cookbooks/private-chef/libraries/preflight_checks.rb

```diff
@@ -28,6 +28,17 @@ def fail_with error
     end
   end
 
+  def first_run?
```

Comment:
Minor style point.  This method could be:

```
def first_run?
  previous_run.nil?
end
```

which would be more in line with the ruby style we lean towards.